Objects/longobject.c

```diff
@@ -5520,6 +5513,8 @@ PyLong_Fini(void)
     /* Integers are currently statically allocated. Py_DECREF is not
        needed, but Python must forget about the reference or multiple
        reinitializations will fail. */
+    Py_DECREF(_PyLong_One);
+    Py_DECREF(_PyLong_Zero);
```

Comment:
Py_CLEAR() would be safer if PyLong_Init() is called again (Py_Initialized called multiple times), no?